spec/haskell/src/SEL4/Kernel/Thread.lhs

```diff
@@ -201,6 +201,11 @@ Replies sent by the "Reply" and "ReplyRecv" system calls can either be normal IP
 
 > doReplyTransfer :: PPtr TCB -> PPtr Reply -> Bool -> Kernel ()
 > doReplyTransfer sender reply grant = do
+>     stateAssert sym_refs_asrt
+>         "Assert that `sym_refs (state_refs_of' s)` holds"
+>     stateAssert cur_tcb'_asrt
+>         "Assert that `cur_tcb' s` holds"
+>     stateAssert weak_sch_act_wf_asrt "assert that `weak_sch_act_wf` holds"
```

Comment:
can't we save space, fit each of these on one line by saying, for example:
```
stateAssert sym_refs_asrt "`sym_refs (state_refs_of' s)`"
```
which says the same thing? `stateAssert X` is literally "assert that X holds"

Reply:
I could do that. Would you like me to do that for all of the `stateAssert`s?

Reply:
These lines were copied and pasted from other functions, since these asserts are old ones. So if I changed these but left the others, it becomes a bit out of sync. I could always merge these PRs with the current version of these `stateAssert`s and then make a separate PR that changes all of them, if people would like that.